src/test/resources/test_data/calculation_breakdown_response/custom-examples/crs-2047-sds-40-labels-5.json

```diff
@@ -55,7 +55,7 @@
       "unadjustedDate": "2027-04-22"
     },
     "CRD": {
-      "rules": ["SDS_EARLY_RELEASE_APPLIES"],
+      "rules": ["SDS_STANDARD_RELEASE_APPLIES"],
```

Comment:
checked with the analyst (Cherky) on this and this test was wrong; i.e. it should actually be expecting SDS_STANDARD_RELEASE_APPLIES